src/server/templates/target.js

```diff
@@ -0,0 +1,28 @@
+import favicon from './favicon';
+
+
+export default ({title, markup, page, props, script}) => `
+	  <!DOCTYPE html>
+	  <html>
+		<head>
+		  <title>${title ? `'${title} – BookBrainz'` :
+		     'BookBrainz – The Open Book Database'}</title>
+		  <link rel="stylesheet" href='/stylesheets/style.css' />
+		  <link rel="stylesheet" href='/stylesheets/react-virtualized.css' />
+		  <link rel="stylesheet"
+		    href='/stylesheets/react-virtualized-select.css' />
+		  <meta name='viewport' content='//react-virtualized.css' />
```

Comment:
Please fix the `content` value for this meta tag. Your use of quotations is also inconsistent, please stick to single-quotes since it is enforced in the rest of the project.